src/main.rs

```diff
@@ -69,7 +70,8 @@ fn run() -> FormatterResult<()> {
     let query_path = if let Some(query) = args.query {
         query
     } else if let Some(language) = args.language {
-        str::to_lowercase(format!("languages/{language:?}.scm").as_str())
+        PathBuf::from(option_env!("TOPIARY_LANGUAGE_DIR").unwrap_or("language"))
```

Comment:
Should be `unwrap_or("languages")`, right?

Reply:
fixed it